src/script/team/TeamService.ts

```diff
@@ -67,6 +67,7 @@ export class TeamService {
   }
 
   getAllTeamFeatures(): Promise<FeatureList> {
+    throw new Error('kaputt');
```

Comment:
Broken?

Reply:
Ooopsie ;P That was for testing the error :P